ui/src/components/NavigationBar/MainMenu.js

```diff
@@ -0,0 +1,415 @@
+              </Link>
+              <Link href="/">
+                <h3>Our values</h3>
+              </Link>
+              <Link href="/">
+                <h3>Statement of Faith</h3>
+              </Link>
+              <Link href="/">
+                <h3>Bold Vision</h3>
+              </Link>
+              <Link href="/">
+                <h3>Harvest Mission International</h3>
+              </Link>
+            </VStack>
+            <VStack
+              alignItems="flex-start"
+              position="relative"
+              left={{ md: "10%", lg: "30%", xl: "40%" }}
+            >
+              <h1>Connect</h1>
+              <Link href="/">
+                <h3>Ministries</h3>
+              </Link>
+              <Link href="/">
+                <h3>LIFE Groups</h3>
+              </Link>
+              <Link href="/">
+                <h1>Events</h1>
+              </Link>
+              <Link href="/">
+                <h1>Sermons</h1>
+              </Link>
+              <Link href="/">
+                <h1>Give</h1>
+              </Link>
+            </VStack>
+          </Flex>
+          <Flex position="relative" left={{ md: "10%", lg: "30%", xl: "40%" }}>
+            <VStack className="third-column-mainmenu" alignItems="flex-end">
+              <Box className="sign-up-button" position="relative">
+                {login ? (
+                  <ButtonGroup
+                    position="relative"
+                    top="20%"
+                    right="30%"
+                    flexDirection="column"
+                    size="md"
+                    variant="outline"
+                    colorScheme="black"
+                    width="200px"
+                  >
+                    <Button marginBottom="5" width="100%">
+                      My Profile
+                    </Button>
+                    <Button marginBottom="5" width="100%">
+                      Log Out
+                    </Button>
+                    <Button width="100%">Prayer Requests</Button>
+                  </ButtonGroup>
+                ) : (
+                  <ButtonGroup
+                    flexDirection="column"
+                    position="relative"
+                    right="30%"
+                    marginTop="20px"
+                    size="md"
+                    variant="outline"
+                    colorScheme="black"
+                    width="200px"
+                  >
+                    <Button marginBottom="5" width="100%">
+                      Login or Sign up
+                    </Button>
+                    <Button width="100%">Prayer Requests</Button>
+                  </ButtonGroup>
+                )}
+              </Box>
+
+              <Box
+                className="other-links"
+                position="relative"
+                top="15%"
+                right="43%"
+              >
+                <VStack alignItems="center" fontSize="sm">
+                  <h2>Ann Arbor</h2>
```

Comment:
aren't these supposed to be outlinks? also why are they just `<h2>` elements instead of using Chakra UI elements

Reply:
changed to links and linked to the respective hmcc locations